HW_1/task3.py

```diff
@@ -0,0 +1,33 @@
+# Задача 3
+
+# Напишите код, который запрашивает число и сообщает является ли оно
+# простым или составным. Используйте правило для проверки:
+# “Число является простым, если делится нацело только на единицу и на себя”.
+# Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+
+# Делаем бесконечный цикл
+while True:
+    num = input("Введите число от 2 до 100000: ")
+    # Делаем обработчик ошибки ввода
+    try:
+        num = int(num)
+        # Ставим ограничение на ввод числа
+        if num < 2 or num > 100000:
+            print("Ошибка ввода")
+        # Если число ввели правильно то проверяем его на "простоту"
+        else:
+            # Предпологаем что число простое пока не докажем обратное
+            is_prime = True
+            # Циклом проходим от 2 до середины введеного числа+1
+            # (любое число, большее чем половина "num" не может быть его делителем без остатка)
+            for i in range(2, int(num/2)+1):
```

Comment:
Во-первых, зачем `int`, если оператор целочисленного деления сделает то же самое?
```suggestion
            for i in range(2, num // 2 + 1):
```
Во-вторых, можно понизить количество итераций чуть сильнее. Вспомните теорию чисел